code/modules/projectiles/projectile/bullets.dm

```diff
@@ -62,13 +62,15 @@
 	fire_sound = 'sound/weapons/Gunshot_c20.ogg'
 
 /obj/item/projectile/bullet/midbullet/lawgiver
-	damage = 10
+	damage = 15
 	stun = 0
 	weaken = 0
 	superspeed = 1
 
 /obj/item/projectile/bullet/midbullet2
-	damage = 25
+	damage = 20
+	stun = 0
+	weaken = 0
```

Comment:
Are you aware that by altering the values of `/obj/item/projectile/bullet/midbullet2` directly instead of making a new child, you're affecting every gun that already uses that bullet?